### DISCUSSION TITLE
Possible to map ABXY to rear buttons?

### DISCUSSION THREAD

**Discussion by swTinker** [Valkirie/HandheldCompanion#327] [Q&A]
Hi Dev!

Wow!!! Thanks for the great app! I've a question, is there a way to map ABXY to rear SteamDeck buttons?

Best,

**Comment by Valkirie** [comment]
Hey, I'm starting mapper development today. Stay tuned.

**Reply by swTinker** [reply]
Thanks for the quick reply.
Accidently stumble onto this tool, love the quick tool menu and overall UI! Makes my on-the-go gaming life soooo much easier!!! Thank you so such 🤝

**Comment by Valkirie** [comment]
Great! Feature should be ready by end of week next week.

**Reply by swTinker** [reply]
Hi! I noticed there's an update yesterday and after fumbling for nearly an hour, I couldn't find a way to map rear buttons, can you share some lights?
Best,

**Comment by CasperH2O** [comment]
@swTinker the recent update was a bugfix and QoL release. As noted in the release notes, work on the remapper is still ongoing. 🙂

**Comment by CasperH2O** [comment]
It's been 6+ months since the Profile Layout feature has been implemented, it's been tested and updated with fixes and new features along the way. Closing this discussion as answered, feel free to reopen if you still have questions. @swTinker
